### ISSUE TITLE
Search for modules listed in `provides` section in `META6.json`

### ISSUE THREAD

**Issue by hythm7** [chloekek/crai#15]
Is it planned to support searching for modules provided by the main module?

for example if I search for `Cro::Uri` (which is listed in `provides` section of `Cro::Core`) Would it return empty result or `Cro::Core`.

**Comment by chloekek** [comment]
Yep this would be a nice feature to have. The first step would be to store the list of modules in the database.

**Comment by chloekek** [comment]
CRAI now stores provides in the database, but this is not yet used for searching nor displayed.
